HTTPServerProject.Tests/HTTPServerProxyTests/TestClient.cs

```diff
@@ -0,0 +1,16 @@
+using HTTPServerProxy.Client;
+
+namespace HTTPServerProxyTests.Client;
+
+public class TestProxyClient : IProxyClient
```

Comment:
See my broader note attached to the bottom about `IProxyClient`

Reply:
I am going to remove the empty interface `IProxyClient` and this unneccessary test double